src/components/templates/ArtPiece/ArtPiece.tsx

```diff
@@ -35,8 +34,7 @@ export interface ArtPieceProps {
 
 export const ArtPiece: React.FC<ArtPieceProps> = ({ venue }) => {
   // NOTE: venue should always be there, but per the if(!venue) check bellow, better make safe than sorry
-  const { name, host, config, showRangers, iframeUrl, videoAspect } =
-    venue ?? {};
+  const { name, host, config, iframeUrl, videoAspect } = venue ?? {};
```

Comment:
looks like `showRangers` is a legacy feature. Please, remove it everywhere from the code.

Reply:
@sunny-viktoryia fixed in d4dcd6783fd04b2501d6998a02a6d4f38db51acb